datadog_checks_dev/datadog_checks/dev/_env.py

```diff
@@ -135,6 +135,15 @@ def replay_check_run(agent_collector, stub_aggregator, stub_agent):
                     data.get('device'),
                 )
 
+        network_devices_events = aggregator.get('network-devices-metadata') or []
+        for data in network_devices_events:
+            stub_aggregator.submit_network_devices_metadata_e2e(
+                check_name,
```

Comment:
We don't need check_name and check_id, do we ?

Reply:
We don't really need them, but all the signature for `aggregator.submit_*` contain check_name and check_id, so I followed a similar signature for consistency.

Note, unlike other methods submit_* method, there is no real agent aggregator binding method that is exposed.